docs/reference/debugging.md

```diff
@@ -306,7 +306,7 @@ If the generated Envoy configuration is not looking as expected, you can manuall
 
 ```shell
 $ kubectl scale deployment ambassador --replicas=1
-deployment.extensions "ambassador" scaled
```

Comment:
Just to doublecheck, this is correct?

Reply:
Yes this is correct. Ambassador Deployments are no longer a `deployment.extension`. They are now a `deployment.apps`. It should be `deployment.apps/ambassador` however so I will quick make that change.